File: models/project.js
const mongoose = require('mongoose');

const ProjectSchema = new mongoose.Schema({
    name: String,
    tag_line: String,
    url_github: String,
    url_live:String,
    can_show: {
        type: Boolean,
        default: true
    }
});

module.exports = mongoose.model('Project', ProjectSchema);

File: Data/Project.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace PortfolioSiteAPI.Data
{
    public class Project
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Tag_Line { get; set; }
        public string Url_Github { get; set; }
        public string Url_Live { get; set; }
        public bool Can_Show { get; set; }
        public ApplicationUser ApplicationUser { get; set; }
        public string ApplicationUserId { get; set; }
    }
}
